### ISSUE TITLE
Generator: Handle Norwegian characters 

### ISSUE THREAD

**Issue by bjornna** [ehrbase/openEHR_SDK#103]
## Background ##

OPTs created for Norwegian language may contain characters like æ,ø.å. The should be replaced with: 

æ -> ae
ø -> oe
å -> aa

I suggest adding a method like the following, or similar functionality. 

```
import com.google.common.base.CharMatcher;
private static String norwegianNormalize(String s){
        s = CharMatcher.is('æ').replaceFrom(s, "ae");
        s = CharMatcher.is('ø').replaceFrom(s, "oe");
        s = CharMatcher.is('å').replaceFrom(s, "aa");
        return s;
    }
```
## Acceptance criteria ##
- [ ] Element with name Testæøå should be Testaeoeaa 

## Success criteria ##

- [ ] Review / Merge request approved (P2P session)
- [ ] Unit tests passed
- [ ] Updated documentation (Javadoc and Sphinx)
- [ ] Acceptance criteria fulfilled
- [ ] Build without errors
- [ ] Release notes prepared
- [ ] Runtime warnings

**Comment by stefanspiska** [comment]
@bjornna @birgerhaarbrandt 
I will in the near future do a refactoring of the generator. And then the name generation (and more)  will configurable.

@bjornna 
I hope that for now it is ok if you simple use you fix in your local build.

**Comment by stefanspiska** [comment]
@bjornna 
Fixed in #153 which will become release 1.0.0
see
https://github.com/ehrbase/openEHR_SDK/blob/release/v1.0.0/CHANGELOG.md for changes
https://github.com/ehrbase/openEHR_SDK/blob/release/v1.0.0/UPDATING.md for how to update

You can define replacements via 
org.ehrbase.client.classgenerator.ClassGeneratorConfig.replaceChars
or by implementing your own
org.ehrbase.client.classgenerator.NamingStrategy
